frontend/packages/schema/src/operation/schema/extension.ts

```diff
@@ -0,0 +1,41 @@
+import * as v from 'valibot'
+import { extensionSchema } from '../../schema/schema.js'
+import { PATH_PATTERNS } from '../constants.js'
+
+// Add extension operation
+const addExtensionOperation = v.object({
+  op: v.literal('add'),
+  path: v.pipe(
+    v.string(),
+    v.regex(PATH_PATTERNS.EXTENSION_BASE),
+    v.description('Path to add extension (e.g., /extensions/vector)'),
+  ),
+  value: extensionSchema,
+})
+
+// Remove extension operation
+const removeExtensionOperation = v.object({
+  op: v.literal('remove'),
+  path: v.pipe(
+    v.string(),
+    v.regex(PATH_PATTERNS.EXTENSION_BASE),
+    v.description('Path to remove extension (e.g., /extensions/vector)'),
+  ),
+})
+
+// Replace extension operation
+const replaceExtensionOperation = v.object({
+  op: v.literal('replace'),
+  path: v.pipe(
+    v.string(),
+    v.regex(PATH_PATTERNS.EXTENSION_BASE),
+    v.description('Path to replace extension (e.g., /extensions/vector)'),
+  ),
+  value: extensionSchema,
+})
```

Comment:
Perhaps this operation should support `ALTER EXTENSION`, but this is an advanced feature. I think it is better not to add it now. What do you think?

https://www.postgresql.org/docs/current/sql-alterextension.html

Reply:
@MH4GF 
Is it okay to merge this PR? (Since there are subsequent PRs)🙏

Reply:
@MH4GF 
Oh, I'm sorry, I didn't understand. I'll fix it!!

Reply:
Thanks!!